### PR TITLE
Roe 2608 Update public register information on the cmo page

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/ROE-2608

### Change description
Added new template as the cmo page now has different public information text to the other pages.

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/managing-officer-corporate.html [hunk 1]
```diff
@@ -75,8 +75,12 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
         name: "contact_email",
         value: contact_email
       }) }}
-
-      {% include "includes/inset-text.html" %}
+      
+      {{ govukInsetText({
+        html: '<h2 class="govuk-heading-m">What information we’ll show on the public register</h2>
```
Comment (mattch1) [inline views/includes/page-templates/managing-officer-corporate.html:80]: Kept is as 
"What information we’ll show on the public register" 
not
"What information we'll shown on the public register"
as shown in the prototype.

**Comment by markpit** [comment]
It looks like this change will also cover [UAR-1582](https://companieshouse.atlassian.net/browse/UAR-1582) (Update and Remove) as I can see changes to the update test.

File: test/__mocks__/text.mock.ts [hunk 2]
```diff
@@ -152,6 +152,7 @@ export const INFORMATION_SHOWN_ON_THE_PUBLIC_REGISTER = "What information we’l
 export const NOT_SHOW_BENEFICIAL_OWNER_INFORMATION_ON_PUBLIC_REGISTER = "We will not show the beneficial owner’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.";
 export const NOT_SHOW_MANAGING_OFFICER_INFORMATION_ON_PUBLIC_REGISTER = "We will not show the managing officer’s home address unless it is the same as their correspondence address. We’ll only show the month and year of their date of birth.";
 export const SHOW_INFORMATION_ON_PUBLIC_REGISTER = "We will show all the information you provide on this screen on the public Register of Overseas Entities.";
+export const SHOW_CMO_INFORMATION_ON_PUBLIC_REGISTER = "We will show all the other information you provide on this screen on the public Register of Overseas Entities.";
```
Comment (mattch1) [inline test/__mocks__/text.mock.ts:155]: Already present, so will remove and change for the const already there then introduce NOT_SHOW_NAME_OR_EMAIL_ON_PUBLIC_REGISTER = "We will not show the name or email address of the person who can be contacted about this managing officer.";
which I will add to the tests.

Comment (mattch1) [reply test/__mocks__/text.mock.ts]: Updated this to use existing line and test the new line.

**Review by dlloyd1-ch — APPROVED** [review]

File: views/includes/page-templates/managing-officer-corporate.html [hunk 3]
```diff
@@ -75,8 +75,12 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
         name: "contact_email",
         value: contact_email
       }) }}
-
-      {% include "includes/inset-text.html" %}
+      
+      {{ govukInsetText({
```
Comment (dlloyd1-ch) [inline views/includes/page-templates/managing-officer-corporate.html:79]: Jut noticed that in your other PR for update remove (UAR01582) that you have a new file for this section:` insert-text-cmo.html `that could be used here.

**Comment by mattch1** [comment]
Closing this PR, will just make sure all changes required for this have been covered by UAR-1582

**Pull request closed without merging**